.circleci/config.yml

```diff
@@ -125,12 +125,14 @@ orbs:
           - run:
               name: Install uv
               command: |
+                echo 'export UV_INSTALL_DIR=$HOME/.uv' >> $BASH_ENV
+                source $BASH_ENV
                 curl -LsSf https://astral.sh/uv/install.sh | sh
-                echo 'source $HOME/.cargo/env' >> $BASH_ENV
+                echo 'source $UV_INSTALL_DIR/env' >> $BASH_ENV
```

Comment:
Does this make it into the user's $PATH?

Reply:
Oh wait, nevermind, this tells the executor to source this file, which presumably sets the PATH appropriately 👍

Reply:
yeah, this should be the special circleci file that gets auto-loaded for you in each `step`